### PR TITLE
[3128625539] Feature/inline code docs

### PR DESCRIPTION
**Pull request by HarrisonHough**
## Description

-  This is a first pass on adding inline code docs (XML commentation) into the Avatar loader module. 
- I tried to stick to the conventions/practices outline here https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments

#### Added

-   Inline code docs

#### Updated

-   A number of functions and properties were renamed or refactored.

#### Removed

-  redundant or unused code, parameters and classes.

## How to Test

-   Add steps to locally test these changes

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.
-   [ ] QA Testing is updated.

## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.

### CODE REVIEW

File: Editor/Module Management/ModuleUpdater.cs [hunk 1]
```diff
@@ -69,14 +87,21 @@ public static void GetCurrentRelease()
...
+        /// </summary>
+        /// <param name="packageName">The name of the Unity package.</param>
+        /// <param name="currentVersion">The current version of the package.</param>
+        /// <param name="latestVersion">The new version of the package.</param>
+        /// <param name="packageUrl">The Git URL of the Unity package.</param>
         private static void PromptForUpdate(string packageName, Version currentVersion, Version latestVersion,
             string packageUrl)
         {
-            bool shouldUpdate = EditorUtility.DisplayDialog("Update Packages",
+            var shouldUpdate = EditorUtility.DisplayDialog("Update Packages",
```
Comment (gadamyan) [inline Editor/Module Management/ModuleUpdater.cs:104]: IT's a bit weird, in the same PR you changed types to vars and vise-versa. What's the standard for Unity?

File: Editor/Module Management/ModuleList.cs [hunk 2]
```diff
@@ -46,14 +56,18 @@ public static class ModuleList
             version = "4.0.2"
         };
 
+        /// <summary>
+        ///     Get installed modules from Modules list.
+        /// </summary>
+        /// <returns>A <c>Dictionary&lt;string, string&gt;</c> in the format of <c>Dictionary&lt;name, version&gt;</c>. </returns>
```
Comment (gadamyan) [inline Editor/Module Management/ModuleList.cs:62]: Is this line correct? Should we use such symbols in the code?

Comment (HarrisonHough) [reply Editor/Module Management/ModuleList.cs:62]: It's correct, perhaps unnecessary but it allows you to show the type of dictionary (including the < and > symbols in the code docs 
image

Comment (HarrisonHough) [reply Editor/Module Management/ModuleList.cs]: I've updated to this now actually 
image

File: Editor/EditorWindowBase.cs [hunk 3]
```diff
@@ -24,38 +28,35 @@ public class EditorWindowBase : EditorWindow
...
         private bool windowResized;
-        
+
 #if !DISABLE_AUTO_INSTALLER
         public static readonly string ERROR_IMAGE_PATH = "Packages/com.readyplayerme.core/Editor/error.png";
 #else
         public static readonly string ERROR_IMAGE_PATH = "Assets/Ready Player Me/Core/Editor/error.png";
 #endif
 
+        /// <summary>
```
Comment (srcnalt) [inline Editor/EditorWindowBase.cs:38]: empty summary

**Review by srcnalt — APPROVED** [review]

**Pull request merged**